actioncable/lib/action_cable/subscription_adapter/postgresql.rb

```diff
@@ -12,16 +12,16 @@ def initialize(*)
 
       def broadcast(channel, payload)
         with_connection do |pg_conn|
-          pg_conn.exec("NOTIFY #{pg_conn.escape_identifier(channel)}, '#{pg_conn.escape_string(payload)}'")
+          pg_conn.exec("NOTIFY #{pg_conn.escape_identifier(channel.to_s)}, '#{pg_conn.escape_string(payload)}'")
```

Comment:
I also looked into using the implicit conversion directly `to_str`. Symbol doesn't implement that, so if we want it to fail for very nearly non-string types, this will work, but I'll need to update the PR and test.